src/main/java/org/spongepowered/common/data/SpongeDataRegistry.java

```diff
@@ -206,6 +202,11 @@ public static void finalizeRegistration() {
         }
         checkArgument(!immutableProcessorList.contains(processor), "Duplicate DataProcessor Registration!");
         immutableProcessorList.add(processor);
+        
+        SpongeSerializationService service = (SpongeSerializationService) Sponge.getGame().getServiceManager().provide(SerializationService.class).get();
```

Comment:
I don't see any work-around. Would moving this to `finalizeRegistration()` fix this?